src/rest_framework_jwt/settings.py

```diff
@@ -43,6 +44,8 @@
     'JWT_AUTH_COOKIE_SECURE': True,
     'JWT_AUTH_COOKIE_SAMESITE': 'Lax',
     'JWT_IMPERSONATION_COOKIE': None,
+    'JWT_DELETE_STALE_BLACKLISTED_TOKENS': False,
+    'JWT_STALE_BLACKLISTED_TOKEN_EXPIRATION_TIME': timezone.now(),
```

Comment:
This will be evaluated to the server's startup time, not dynamically evaluated. Try using `timezone.now` which sets it to a function instead.